packages/dds/tree/src/serializableDomainSchema.ts

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IFluidHandle } from "@fluidframework/core-interfaces";
+import {
+	SchemaFactory,
+	type FixRecursiveArraySchema,
+	type TreeNodeFromImplicitAllowedTypes,
+	type ValidateRecursiveSchema,
+} from "./simple-tree/index.js";
+import type { JsonCompatible } from "./util/index.js";
+
+const sf = new SchemaFactory("com.fluidframework.serializable");
+
+/**
+ * Utilities for storing {@link FluidSerializableAsTree.Data|Fluid Serializable data} in {@link TreeNode}s.
+ *
+ * Same as {@link JsonAsTree} except allows {@link @fluidframework/core-interfaces#(IFluidHandle:interface)}s.
+ * @remarks
+ * Schema which replicates the Fluid Serializable data model with {@link TreeNode}s.
```

Comment:
This is intended to be "Schema" plural, right? So, this should be:
```suggestion
 * Schema which replicate the Fluid Serializable data model with {@link TreeNode}s.
```
?